packages/core/test/acceptance/pipe_spec.ts

```diff
@@ -706,128 +707,156 @@ describe('pipe', () => {
     });
   });
 
-  describe('missing pipe detection logic', () => {
-    it('should throw an error if a pipe is not found in a component', () => {
-      @Component({
-        template: '{{ 1 | testMissingPipe }}',
-      })
-      class TestComponent {
-      }
-
-      TestBed.configureTestingModule({declarations: [TestComponent]});
-
-      expect(() => {
-        const fixture = TestBed.createComponent(TestComponent);
-        fixture.detectChanges();
-      })
-          .toThrowError(
-              /The pipe 'testMissingPipe' could not be found in the 'TestComponent' component!/);
-    });
-
-    it('should throw an error if a pipe is not found inside an inline template', () => {
-      @Component({
-        template: `
-          <ng-container *ngIf="true">
-            {{ value | testMissingPipe }}
-          </ng-container>`
-      })
-      class TestComponent {
-        value: string = 'test';
-      }
-
-      TestBed.configureTestingModule({declarations: [TestComponent]});
-
-      expect(() => {
-        const fixture = TestBed.createComponent(TestComponent);
-        fixture.detectChanges();
-      })
-          .toThrowError(
-              /The pipe 'testMissingPipe' could not be found in the 'TestComponent' component!/);
-    });
-
-    it('should throw an error if a pipe is not found inside a projected content', () => {
-      @Component({selector: 'app-test-child', template: '<ng-content></ng-content>'})
-      class TestChildComponent {
-      }
-
-      @Component({
-        template: `
-          <app-test-child>
-            {{ value | testMissingPipe }}
-          </app-test-child>`
-      })
-      class TestComponent {
-        value: string = 'test';
-      }
-
-      TestBed.configureTestingModule({declarations: [TestComponent, TestChildComponent]});
-
-      expect(() => {
-        const fixture = TestBed.createComponent(TestComponent);
-        fixture.detectChanges();
-      })
-          .toThrowError(
-              /The pipe 'testMissingPipe' could not be found in the 'TestComponent' component!/);
-    });
+  [false, true].forEach(componentIsStandalone => {
+    const expectedThrowRegex = new RegExp(
+        'The pipe \'testMissingPipe\' could not be found in the \'TestComponent\' component,' +
+        (componentIsStandalone ? ' verify that it is imported by the component' :
+                                 ' verify that it is declared or imported in this module'));
```

Comment:
nit: I like the phrasing that we've used in other error messages, which specifically mentions `@Component.imports` for standalone case, which gives a bit more info to users:
```suggestion
        (componentIsStandalone ? ' verify that it is included in the \'@Component.imports\' of this component' :
                                 ' verify that it is declared or imported in this module'));
```